wp-content/themes/core/components/header/subheader/Subheader_Controller.php

```diff
@@ -61,15 +61,16 @@ protected function defaults(): array {
 
 	protected function required(): array {
 		return [
-			self::MEDIA_CLASSES   => [ 'c-subheader__media' ],
-			self::CONTENT_CLASSES => [ 'c-subheader__content' ],
-			self::CLASSES         => [ 'c-subheader'],
+			self::CONTAINER_CLASSES => [ 'l-container' ],
+			self::MEDIA_CLASSES     => [ 'c-subheader__media' ],
+			self::CONTENT_CLASSES   => [ 'c-subheader__content' ],
+			self::CLASSES           => [ 'c-subheader', 'c-subheader--has-background'],
```

Comment:
Not sure we want to force `c-subheader--has-background`, do we? This seems like one we'd want to add in the `get_classes()` method.

Reply:
Ah good call. Fixed.